docs/source/app-dev/app-arch.rst

```diff
@@ -148,6 +148,24 @@ To use command deduplication, you should:
 
 For more details on command deduplication, see the :ref:`Ledger API Services <command-submission-service-deduplication>` documentation.
 
+
+.. _handling-participant-node-failover:
+
+Handling participant node failover
+**********************************
+
+Some DAML Ledgers support exposing multiple eventually consistent Ledger API endpoints hosted by separate participant nodes.
+
+To support switching to a different participant node at run time,
```

Comment:
s/participant node/Ledger API endpoint

Reply:
these terms are not interchangeable. A second Ledger API endpoint can be backed by the same index DB (participant) and therefore won't have "eventual consistency" problems.

Reply:
@gerolf-da : yes they won't have these problems, but they will be eventually consistent. So they do satisfy the assumptions here.